specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/secrets.json

```
@@ -572,6 +572,137 @@
           }
         }
       }
+    },
+    "/secrets/{secret-name}/rotationpolicy": {
+      "get": {
+        "tags": [
+          "Secrets"
+        ],
+        "operationId": "GetSecretsRotationPolicy",
+        "summary": "Lists the policy for a secret.",
+        "description": "The GetSecretRotationPolicy operation returns the specified secret policy resources in the specified secret vault. This operation requires the secret rotation policy get permission.",
+        "parameters": [
+          {
+            "name": "secret-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the secret in a given key vault."
```

Should capitalize product names, or maybe just say "vault" here.
```suggestion
            "description": "The name of the secret in a given Key Vault."
```

---

Updated with capitalized product names.